common/views/components/Footer/FooterSocial.tsx

```diff
@@ -124,7 +124,7 @@ const FooterSocial: FC = () => (
       <Cell key={item.title}>
         <Link href={item.url}>
           <Space as="span" h={{ size: 's', properties: ['margin-right'] }}>
-            <Icon icon={item.icon} color="green" />
+            <Icon icon={item.icon} color="currentColor" />
```

Comment:
Did this as discussed the Icons being green with Gareth and that's what's sparked the eventual refactor of the Footer we're going to do; eventually icons will be set to `currentColor` by default as well.